src/middelware/legacy_event_to_cloudevent.ts

```diff
@@ -0,0 +1,218 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import {Request, Response, NextFunction} from 'express';
+import * as cloudevents from '../cloudevents';
+import {CE_TO_BACKGROUND_TYPE} from './ce_to_legacy_event';
+import {CloudFunctionsContext, LegacyEvent} from '../functions';
+
+const BACKGROUND_TO_CE_TYPE = new Map<string, string>(
```

Comment:
Can we use a raw object rather than a `new Map<string, string>`? We can add the type interface in the const.